src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -155,9 +145,10 @@ private void configureAutoBuilder() {
    * @return the current speed of the drivetrain
    */
   public ChassisSpeeds getSpeeds() {
-    return m_relativeSpeeds;
+    return m_kinematics.toChassisSpeeds(new SwerveModuleState[] {m_frontLeft.getState(), m_frontRight.getState(), m_rearLeft.getState(), m_rearRight.getState()});
```

Comment:
I recommend putting this into an array and using that instead with a for loop as it'll be cleaner.